docs/reference/modules/search.md

```diff
@@ -30,6 +30,15 @@ modules/@apostrophecms/page/index.js
 </template>
 </AposCodeBlock>
 
+## Customizing the search template
+
+The `@apostrophe/search` module template can be customized through adding files to the project-level `modules/@apostrophecms/search/views` folder. The module has two templates for returning search results. Depending on your site requirements you may only need to add a single template at the project level.
+
+The default template is `index.html`. The main block of this template contains a form for adding any filters to the search if the `filters` option has been populated, as well as the query terms. Below the form is a section to output the search results returned to the page in the `data.doc` object. Finally, the page includes the [standard pagination section](/guide/piece-pages.md#pagination). In this case, it is added using `{% include "pager.html" %}` to load it in from another file in the `views` folder so that it can be easily reused. You can elect to add the pager directly to your project template or keep it as a separate file.
+
+The `indexAjax.html` template will be displayed to the user if `req.xhr` or `req.query.xhr` are set to indicate that an AJAX request is being made. This template only contains the markup for displaying the information contained within the `data.docs` object returned from the search query.
```

Comment:
You know what, let's not talk about the `indexAjax.html` template. We don't actually have any javascript in A3 at the moment that would ever invoke it, and it would kickstart a conversation about why we don't have instant navigation for A3 yet. I'm still in the process of negotiating that at a strategic level. Also the term "ajax" is a very old brand so we should be thinking about stomping it out.